app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -80,7 +80,7 @@ import java.io.IOException
 class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
     private val b by viewBinding(ActivitySettingsScreenBinding::bind)
 
-    private val FILETAG : String = "Settings Fragment-"
+    private val FILETAG: String = "Settings Fragment-"
```

Comment:
Ideally, log tag is `SettingsFragment.getSimpleName()` See: https://stackoverflow.com/questions/15202997

Change for all classes.

```
Log.x(TAG, msg)
// where TAG is a class' simple-name, not RethinkDNS
// logcat already separates out log output by app name
// and so, it needn't be specified as a TAG.
```

Reply:
With the TAG as application name, I can able to segregate the app logs. Android logs is not giving the app name separately. 

_04-13 20:08:28.368  9503  9600 D RethinkDNS: RethinkVPNService isLoadFirewallRulesCompleted checkConnectionBlocked is true
04-13 20:08:28.389  9503  9600 D RethinkDNS: RethinkVPNService : isScreenLockEnabled: false
04-13 20:08:28.391  9503  9579 D RethinkDNS: IPTracker - Package for uid : 10237, 1_

Reply:
> With the TAG as application name, I can able to segregate the app logs. Android logs is not giving the app name separately.

The log is already segregated per-app, using app uid, pid, and package-name. Here's how to  use `adb logcat` with package-name: https://stackoverflow.com/questions/6854127

So: An app-wide tag is redundant. log-tag should be class name instead of being concatenated into the log msg, which looks ugly. Pls get rid.

Reply:
Added the package name as the log-tag. 
`Log.<log level>("<your package name>", "message");`

Reply:
Modified the logs. Added new tags. 

- "NonStoreAppUpdater"
- "FirewallLogs"
- "DnsLogs"
- "AppMode"
- "VpnLifecycle"
- "ConnectivityEvents"
- "DnsManager"
- "FirewallManager"
- "PermissionManager"
- "AppDatabase"
- "DownloadManager"
- "ActivityManager"